src/views/signedExtrinsics/index.tsx

```diff
@@ -0,0 +1,218 @@
+import { useToggleVisibility } from '@pivanov/use-toggle-visibility';
+import { formatDistanceToNowStrict } from 'date-fns';
+import {
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+
+import { Icon } from '@components/icon';
+import { ModalShowJson } from '@components/modals/modalShowJson';
+import { PageHeader } from '@components/pageHeader';
+import { PDScrollArea } from '@components/pdScrollArea';
+import { SearchBar } from '@components/searchBar';
+import { useStoreChain } from '@stores';
+import {
+  cn,
+  truncateAddress,
+} from '@utils/helpers';
+
+import type {
+  IMappedBlockExtrinsic,
+  IMappedTransferExtrinsic,
+} from '@custom-types/block';
+
+const SignedExtrinsics = () => {
+  const blocksData = useStoreChain?.use?.blocksData?.();
+  const chain = useStoreChain?.use?.chain?.();
+  const latestBlock = useStoreChain?.use?.bestBlock?.();
+  const [ShowJsonModal, toggleVisibility] = useToggleVisibility(ModalShowJson);
+
+  const [signedExtrinsics, setSignedExtrinsics] = useState<IMappedTransferExtrinsic[]>([]);
+  const [selectedExtrinsic, setSelectedExtrinsic] = useState<IMappedTransferExtrinsic | null>(null);
+  const refInitalExtrinsicsDisplayed = useRef(false);
+  const isLoading = blocksData.size === 0;
+
+  const filterExtrinsics = useCallback((extrinsics: IMappedBlockExtrinsic[] = []) => {
+    return extrinsics.filter(extrinsic => extrinsic.isSigned).reverse() as IMappedTransferExtrinsic[];
+  }, []);
+
+  const loadInitialData = useCallback(() => {
```

Comment:
Maybe we could think of refactor the way we are getting the extrinsics list, as it is now being used on multiple places and iterating the whole structure in each component could eventually lead to lack of performance.